src/ServiceBouncer/MainForm.cs

```diff
@@ -19,6 +19,10 @@ public partial class MainForm : Form
         private bool isActive;
         private string machineHostname;
         private int backgroundRefreshSeconds;
+        private DateTime? machineInactivatedTime;
+        private int inactivityMinutesUntilAppTermination = 30;
```

Comment:
If you make this a timespan it make the checking easier.  You can say is this timespan bigger that other timespan.